controllers/apply/digital-funding/success.njk

```diff
@@ -4,22 +4,8 @@
 
 {% block title %} {{ title }} | {{ formTitle }} | {% endblock %}
 
-{% if not heroImage %}
-    {% set bodyClass = 'has-static-header' %}
-{% endif %}
-
 {% block content %}
     <main role="main" id="content">
-        {% if heroImage %}
```

Comment:
We no longer use hero images for the digital fund so removing this code completely.